easybuild/easyconfigs/p/partitionfinder/partitionfinder-2.1.1-intel-2019b-Python-2.7.16.eb

```diff
@@ -0,0 +1,49 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Tarball'
+
+name = 'partitionfinder'
+version = '2.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/brettc/partitionfinder/releases/tag/v2.1.1'
+description = """PartitionFinder 2 is a Python program for simultaneously
+choosing partitioning schemes and models of molecular evolution for phylogenetic
+analyses of DNA, protein, and morphological data. You can PartitionFinder 2
+before running a phylogenetic analysis, in order to decide how to divide up
+your sequence data into separate blocks before analysis, and to simultaneously
+perform model selection on each of those blocks."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+github_account = 'brettc'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['ccf3718996ee6ca496909b4b97d2b075028e0543eba3bc47a8c14b689c84e061']
+
+dependencies = [
+    ('Python', '2.7.16'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('scikit-learn', '0.20.4', versionsuffix),
+    ('PyTables', '3.5.2', versionsuffix),
+    ('pyparsing', '2.4.6', versionsuffix),
+]
+
+postinstallcmds = ["cd %(installdir)s && chmod a+x PartitionFinder*"]
+fix_python_shebang_for = ["*.py"]
+
+sanity_check_paths = {
+    'files': ['PartitionFinderMorphology.py', 'PartitionFinderProtein.py', 'PartitionFinder.py'],
+    'dirs': ['partfinder'],
+}
+
+sanity_check_commands = [
+    'PartitionFinder.py %(installdir)s/examples/nucleotide/test.phy',
+    'PartitionFinderProtein.py %(installdir)s/examples/aminoacid/test.phy',
+    'PartitionFinderMorphology.py %(installdir)s/examples/morphology/test.phy --raxml',
```

Comment:
@Darkless012 How long did this last test command take for you?

In my test, it's been running for over 2 hours now (consuming 9 cores), surely that's a bit much for a sanity check... 😅

Reply:
Eh, I'm not sure, but 2hrs doesn't seem right. Maybe it is an overkill, but it is just during the installation once.
Should update eb to keep only one of them?

Reply:
I'm not seeing this issue anymore when trying again now, the whole installation of `PartitionFinder` now completes in about a minute, no sure what was going on before, but I'll blame it on cosmic rays. ;)

Reply:
I can reproduce the issue when I'm performing the PartitionFinder installation in a job that requests a partial node.

As long as the installation is done on a *full* node (all cores requested), it's OK.

Is there a way to tell `PartitionFinder` how many cores it's allowed to use? It seems by default it's assuming it can use all the cores it can see on the system, which seems to rely in some kind of hang when this test is running...

Reply:
Updated the tests. It should be fixed by `-p <INT>` for parallelism hint cli option.